CHANGES.rst

```diff
@@ -5,6 +5,9 @@
 3.5.0a5 (unreleased)
 ====================
 
+- Support using RelStorage in a FIPS enabled environment. See
```

Comment:
I wouldn't claim 'support', because it's not a tested configuration. I can reword the change note though.